README.md

```diff
@@ -1,5 +1,8 @@
 # Microsoft.Build (MSBuild)
 
+[![Build Status](https://dev.azure.com/dnceng-public/public/_apis/build/status/75?branchName=main&label=Public)](https://dev.azure.com/dnceng-public/public/_build?definitionId=75) [![Build Status](https://devdiv.visualstudio.com/DevDiv/_apis/build/status/9434?label=Private&branchName=main)](https://devdiv.visualstudio.com/DevDiv/_build?definitionId=9434)
```

Comment:
I think the wording for private a bit off in this context. Internal sounds a bit better, and it is what other repos, [like SDK](https://github.com/dotnet/sdk?tab=readme-ov-file#build-status), use.

Reply:
Agree! Updated! :) Thank you!